WORKSPACE

```diff
@@ -25,61 +25,30 @@ load(
 )
 
 com_google_gapic_generator_cpp_gax_repositories()
+
+load(
+    "@com_google_googleapis//:repository_rules.bzl",
+    "switched_rules_by_language",
+)
+
+switched_rules_by_language(
+    name = "com_google_googleapis_imports",
+    cc = True,
+    gapic = True,
+    grpc = True,
+)
+
 com_google_gapic_generator_cpp_repositories()
 
 load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
 
 grpc_deps()
 
-# WARNING: The java and go segments below are a hack to get
-# C++ operations proto types to build.
-# Tracked by https://github.com/googleapis/gapic-generator-cpp/issues/35
-
-##############################################################################
-# Java
-##############################################################################
-
 #
-# gapic-generator
+# gapic-generator (contains common definitions for gapic rules)
 #
 http_archive(
```

Comment:
Why do we need this, now that we removed the code below?

Reply:
we still need the gapic generator repo for  `gapic.bzl` file (we do not need the generator itself).